.jenkins/pipelines/Agnostic/Linux/Jenkinsfile

```diff
@@ -116,6 +146,14 @@ try{
                 "Sim 1804 clang-10 SGX1FLC RelWithDebInfo":          { simulationContainerTest('18.04', 'RelWithDebInfo', ['-DHAS_QUOTE_PROVIDER=ON',  '-DLVI_MITIGATION=None', '-DLVI_MITIGATION_SKIP_TESTS=OFF']) },
                 "Sim 1804 clang-10 SGX1FLC Debug snmalloc":          { simulationContainerTest('18.04', 'Debug',          ['-DHAS_QUOTE_PROVIDER=ON',  '-DLVI_MITIGATION=None', '-DLVI_MITIGATION_SKIP_TESTS=OFF', '-DUSE_SNMALLOC=ON']) },
                 "Sim 1804 clang-10 SGX1FLC RelWithDebInfo snmalloc": { simulationContainerTest('18.04', 'RelWithDebInfo', ['-DHAS_QUOTE_PROVIDER=ON',  '-DLVI_MITIGATION=None', '-DLVI_MITIGATION_SKIP_TESTS=OFF', '-DUSE_SNMALLOC=ON']) },
+                "Sim 2004 clang-10 SGX1 Debug":                      { simulationContainerTest('20.04', 'Debug',          ['-DHAS_QUOTE_PROVIDER=OFF', '-DLVI_MITIGATION=None', '-DLVI_MITIGATION_SKIP_TESTS=OFF']) },
```

Comment:
These simulation mode tests on SGX machine seem unrelated to the goal of this PR which is to add cross platform build to CI.

The value of simulation mode is much less on machines with SGX support; the number of these configurations can be reduced for each CI run.

Reply:
I was adding these for 18.04 and 20.04 parity. Do you recommend distributing different configurations between both OS's instead of duplicating them on both?

Reply:
@mingweishih Is there much value in having so many simulation mode runs given that we test simulation mode already on non-sgx hardware?